.github/workflows/main.yaml

```
@@ -171,26 +181,35 @@ jobs:
       - name: Test
         run: |
           make test
-        if: matrix.target != 'aarch64-apple-darwin'
+        if: matrix.target != 'aarch64-apple-darwin' && matrix.build != 'linux-musl-x64'
```

For the sake of consistency:

```suggestion
        if: matrix.build != 'macos-arm64' && matrix.build != 'linux-musl-x64'
```